### PR TITLE
Move to k8s 1.19 and updates vendors for klogv2

### PR DESCRIPTION
**Pull request by huffmanca**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
As part of the effort to move to klog v2, this updates the vendor dependencies to 1.19 so that they can use klogv2. In addition, it updates the iSCSI driver to use `k8s/utils`, as these were moved out of tree in 1.17.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,46 +6,55 @@ require (
...
+replace (
+	k8s.io/api => k8s.io/api v0.19.4
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.4
+	k8s.io/apiserver => k8s.io/apiserver v0.19.4
+	k8s.io/apimachinery => k8s.io/apimachinery v0.19.4
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.19.4
+	k8s.io/client-go => k8s.io/client-go v0.19.4
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.19.4
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.19.4
+        k8s.io/code-generator => k8s.io/code-generator v0.19.4
```
Comment (msau42) [inline go.mod]: the spacing here seems off?

File: pkg/iscsi/iscsi_util.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/golang/glog"
```
Comment (msau42) [inline pkg/iscsi/iscsi_util.go:24]: Hm we probably want to switch glog to use klog v2. That could be done in a separate pr

File: go.mod (hunk 1 above)
Comment (huffmanca) [reply go.mod]: This should now be addressed.

File: pkg/iscsi/iscsi_util.go (hunk 2 above)
Comment (huffmanca) [reply pkg/iscsi/iscsi_util.go:24]: https://github.com/kubernetes-csi/csi-driver-iscsi/pull/35 has been submitted to move to klog v2.

**Pull request merged**